swagger.yaml

```diff
@@ -1894,7 +1894,56 @@ paths:
         required: true
         type: string
   "/{account}/reports/{report}/runs":
-    x-summary: Report Runs
+    x-summary: Report Run
```

Comment:
Should this still be `Report Runs`?

Reply:
This is for the `POST`, which returns a run, so I think this is ok.

Reply:
Ah, yes, correct. :)

Reply:
Or, wait, on line 1898 it says `get:` -- this file is hard to read so not sure if it is for the above or for something below.